src/cedict-service/service.js

```diff
@@ -14,7 +14,7 @@ const messagingServiceName = 'CedictRequestListener'
  */
 const CedictDestinationConfig = {
   name: 'cedict',
-  targetURL: 'http://data-dev.alpheios.net',
+  targetURL: 'https://lexis-dev.alpheios.net',
```

Comment:
what is the plan for supporting different urls per environment (prod/qa/dev)?

Reply:
That's a really good question! Did not think about it yet, really.

Reply:
Do we want to be able to change URL externally, without rebuilding a LexisCS bundle?

Reply:
Ideally, yes.

Reply:
That is addressed in alpheios-project/components#872 and because the change required is significant I will merge the current build without implementing this feature, as agreed upon a Slack discussion.